src/app/static/src/js/components/runReport/parameterList.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div class="col-sm-6">
+      <table class="table table-sm table-bordered">
+        <tbody>
+        <tr v-for="(param, index) in params" :key="index">
+          <th><label :for="`param-control-${index}`"
+                     class="col-sm-2 col-form-label text-right">
+                     {{param.name}}
+          </label></th>
+          <th><input type="text" class="form-control"
+                     v-model="param.default"
```

Comment:
I think it would be cleaner to have a `param.value` field, initialised to the default value (if any)

Reply:
This is a good idea, I have raised it up with Rob, Once they changed the return object from the server, I will replicate the same. For not, default value is ideal.